examples/vault-ec2-auth/user-data-auth-client.sh

```diff
@@ -0,0 +1,110 @@
+#!/bin/bash
+# This script is meant to be run in the User Data of each EC2 Instance while it's booting. The script uses the
+# run-consul script to configure and start Consul in client mode. Note that this script assumes it's running in an AMI
+# built from the Packer template in examples/vault-consul-ami/vault-consul.json.
```

Comment:
Update comment that this script is meant as an example of what Vault clients do to automatically auth.